.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on:
+    pull_request:
+        branches: ['master']
+
+jobs:
+    run:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/setup-node@v1
+            - name: Install yarn
+              run: npm install -g yarn
+            - name: Checkout BASE
+              uses: actions/checkout@v2
+              with:
+                  ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Hi @bryceosterhaus could you please try a build without this `ref:` line? I suspect it's causing the current Coveralls error:

```
##[error]Error: Command failed: git cat-file -p a5d47edd464307d959f773e645f96e43542acfc2
fatal: Not a valid object name a5d47edd464307d959f773e645f96e43542acfc2
```

Reply:
Ah good call. That seemed to fix it.

Thanks so much for your help!